sqlitecluster/SQLite.cpp

```diff
@@ -438,7 +438,7 @@ bool SQLite::verifyTable(const string& tableName, const string& sql, bool& creat
 }
 
 bool SQLite::verifyIndex(const string& indexName, const string& tableName, const string& indexSQLDefinition, bool isUnique, bool createIfNotExists) {
-    SINFO("Verifying index '" << indexName << "'. isUnique? " << to_string(isUnique));
+    SINFO("Verifying index", {{"indexName", indexName}, {"isUnique?", to_string(isUnique)}});
```

Comment:
This is a simple change to quickly test the change.

Reply:
is this for sure converting to a map? 

asking because we had a situation recently where using `{"", ""}` was creating a string

Reply:
If you try to pass a string, it fails to compile. Also, see my test, it would not work if it was a string 😁